XPloit.Modules/Encoders/Xor.cs

```diff
@@ -0,0 +1,28 @@
+﻿using XPloit.Core;
+using XPloit.Core.Attributes;
+
+namespace Encoders.Xor
+{
+    [ModuleInfo(Author = "Teeknofil", Description = "Encode byte[] to Xor bytes")]
+    public class XorEncoder : XPloit.Core.Encoder
+    {
+        /// <summary>
+        /// Encode
+        /// </summary>
+        /// <param name="bytes"></param>
+        /// <param name="strKey"></param>
+        /// <param name="payload"></param>
+        /// <returns></returns>
+       //TODO: fix Target String
+        public static byte[] Run(byte[] bytes, string strKey, Payload payload)
+        {
+            if (payload == null) return null;
+
+            int amount = 350;
```

Comment:
Why this amout?

Reply:
You can update if you want. Excuse my bad English. Or put algorithm of encryption  for future meterpreter payload  in the framework. Example : AES,DES, Core or Encoder Module ?